server/index.ts

```diff
@@ -67,6 +69,17 @@ async function handleLogin(player: alt.Player, email: string, password: string,
         return;
     }
 
+    if (loggedInPlayers.some(player => player._id === account._id)) {
+        const date = new Date();
+        const timestamp = date.toLocaleString('en-GB')
+        const _id = await db.create({ existing_account_id: account._id, suspicious_ip: player.ip, timestamp }, "AuthPlugin_Logs");
```

Comment:
I don't think it's necessary to have a separate collection for this.

Reply:
Maybe I'll do a separate plugin like you said with more logs about authentication.